tests/model_registry/rbac/test_mr_rbac.py

```diff
@@ -0,0 +1,299 @@
+            model_registry_namespace: The namespace where Model Registry is deployed
+            admin_client: The admin client for accessing the cluster
+            test_idp_user_session: The test user session containing both admin and user contexts
+
+        Raises:
+            AssertionError: If access permissions don't match expectations
+            ForbiddenException: Expected before group addition, unexpected after
+        """
+        LOGGER.info(
+            "-----Test that a user can access to the Model Registry once added to a "
+            "group that has the permissions to access it-----"
+        )
+        assert model_registry_instance.name == MR_INSTANCE_NAME
+        model_registry_users_group = f"{MR_INSTANCE_NAME}-users"
+
+        # Verify initial access denied
+        with use_context(test_idp_user_session.user_context):
+            _, client_args = setup_mr_client(
+                model_registry_instance=model_registry_instance,
+                model_registry_namespace=model_registry_namespace,
+                admin_client=admin_client,
+            )
+
+            with pytest.raises(ForbiddenException) as exc_info:
+                _ = ModelRegistryClient(**client_args)
+            assert exc_info.value.status == 403
+
+        # Add user to group
+        group = Group(
+            client=admin_client,
+            name=model_registry_users_group,
+        )
+        group.update(
+            resource_dict={"metadata": {"name": model_registry_users_group}, "users": [test_idp_user_session.username]}
+        )
+
+        # Verify group membership
+        group = Group(client=admin_client, name=model_registry_users_group)
+        users = group.instance.get("users", []) or []
+        assert test_idp_user_session.username in users, (
+            f"User {test_idp_user_session.username} not in group {model_registry_users_group}. Current users: {users}"
+        )
+        LOGGER.info(f"Added user {test_idp_user_session.username} to {model_registry_users_group} group")
+        try:
+            # Wait for access to be granted
+            with use_context(test_idp_user_session.user_context):
+                _, client_args = setup_mr_client(
+                    model_registry_instance=model_registry_instance,
+                    model_registry_namespace=model_registry_namespace,
+                    admin_client=admin_client,
+                )
+
+                sampler = TimeoutSampler(
+                    wait_timeout=240,
+                    sleep=5,
+                    func=assert_positive_mr_registry,
+                    model_registry_instance=model_registry_instance,
+                    model_registry_namespace=model_registry_namespace,
+                    admin_client=admin_client,
+                )
+                for _ in sampler:
+                    break  # Break after first successful iteration
+                LOGGER.info("Successfully accessed Model Registry")
```

Comment:
Ditto. Please create an utility. Even though it is only used once, it would keep the test code small and concise :).

Reply:
I understand the previous one, but here it is used only once as you said but also it is only calling the other methods `setup_mr_client` and `TimeoutSampler`. IMHO is not worth to create a 3rd utils to just call 2 other utils and nothing more